tox.ini

```diff
@@ -129,6 +129,22 @@ deps =
 commands =
     pytest -v --tb native --log-cli-level=INFO -s {posargs} {[vars]tests_path}/integration/ha/test_ha.py
 
+[testenv:ha-storage-integration]
```

Comment:
Should this be added to CI?

Reply:
Good point, on the CI the storage type is `rootfs` which doesn't allow the detaching / mount etc.. so these tests are automatically skipped in that case. Not having them added to it will prevent us from unnecessarily allocate workers